aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -147,9 +156,36 @@ export const SignUpExample = () => {
               label="Password"
               htmlFor="password-sign-up"
               name="password"
-              help="Include at least 8 characters, a lowercase letter, an
-              uppercase letter, a number, and a special character"
-              validate={passwordRulesStrong}
+              info={
+                <List
+                  data={passwordRules}
+                  border={{ color: 'none' }}
+                  pad="none"
+                >
+                  {rule => {
+                    if (
+                      formValues.password === undefined ||
+                      formValues.password.length === 0
+                    ) {
+                      return (
+                        <Box direction="row" gap="xsmall">
+                          <Text size="xsmall">{rule.message}</Text>
+                        </Box>
+                      );
+                    }
+                    return (
+                      <Box direction="row" gap="xsmall">
+                        {formValues.password && rule.valid ? (
+                          <FormCheckmark color="text-strong" />
```

Comment:
Curious to see what these would look like as a smaller size -- the designs make it look like the icons are smaller

Reply:
smaller size it is! This was approved on Greg as well